scripts/scale.sh

```diff
@@ -325,10 +325,13 @@ analyzer:
     fabric:
 EOF
 
-	TOTAL_AGENT=$(( $AGENT_NUM + $AGENT_STOCK ))
+	TOTAL_AGENT=$(( $AGENT_NUM ))
```

Comment:
the scale test is failing, I didn't check yet why but you shouldn't remove the AGENT_STOCK. It is used to provision agent ports that will be used later by the test in order to start more agents

Reply:
Ack, I didn't understand the meaning of AGENT_STOCK, and it was complicating the displayed graph.